_data/slate.yml

```diff
@@ -1,102 +1,4 @@
 en:
   - master
-  - 1.4
```

Comment:
IMO I don't think end users have enough patience to even consume the existing 'master' documentation leave alone version'ed documentation. With agile approach and lot more frequent releases it does not make sense to maintain so many versions. We are bound to miss doc updates as well.